client.go

```diff
@@ -0,0 +1,93 @@
+package manifestival
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type Client interface {
```

Comment:
Nice, is the plan to write an adaptor for the controller-runtime client to fulfill this interface too? If so: ❤️

Reply:
Yes, I have initial stabs for both client impls at https://github.com/manifestival. I don't love their names, though.